caseworker/core/services.py

```
@@ -234,3 +234,10 @@ def get_menu_notifications(request):
         request.cached_get_menu_notifications = client.get(request, "/gov-users/notifications/")
     response = request.cached_get_menu_notifications
     return response.json()
+
+
+def get_mention_count(request):
+    if not hasattr(request, "cached_get_mention_count"):
+        request.cached_get_mention_count = client.get(request, "/cases/user-case-note-mentions/")
```

Please can there be some response code checking? e.g. fail loudly on non 2XX responses `response.raise_for_status()` would probably do the trick